Oak's Turtle and basic Python Exercise

Blucka12 opened the pull request:
@eah13 @gerbal This is the submission of my turtle exercise, which is admittedly crude and in need of more love. Also included is my python exercise, which is in development and should be ignored for now. Thanks!

Blucka12 committed:
Oak's Turtle Assignment

This was a good learning experience...

Blucka12 committed:
Merge pull request #1 from silshack/gh-pages

Updated Oak's personal site

Blucka12 committed:
Create oakpythonexercise.md

Blucka12 committed:
Oak's Second Assignment

Assignment with all associated code. I enjoyed learning this, and hope to learn how to condense and use functions more soon!

Blucka12 commented:
Ok, @eah13  @gerbal  , the .md entitled "Oak's Second Assignment" is the Assignment due tonight. Hopefully, I can get clear on how to complete this task without causing a mess next time. What do I need to do differently to edit the Pull request I was working on?

gerbal commented:
@Blucka12 a couple of things
First you can combine your two python posts into one, it will make our lives a little easier.  
Second, in your turtle post should be an actual post. Make sure to include the header, name the file consistently, and include a picture of the output.  
Third, put your code in `````code blocks, it's easier to read and MarkDown won't try to format it like normal text.

Blucka12 committed:
Oak Added '''s to help the md out a bit.

I'll see what I need to do to combine my posts...

Blucka12 committed:
Oak fixed the ```s to include all of the code

bah!

Blucka12 committed:
Oak is deleting that which is not the Assignment 2

I don't believe that this code is useful to anyone, therefore it gets the axe...

Blucka12 commented:
Ok @gerbal , I think I deleted the stuff I didn't want to be part of the pull request. I didn't do the pic of the output bit yet and I still have to fix the turtle post. I'll get to that soon.

Blucka12 committed:
Oak updated Turtle Assignment

Titled it like a real post and shook his head in shame at the look of it...wants the turtle-hamster-wheel to be a success...

eah13 commented:
@Blucka12 Did you preview your site when submitting it?  Next time make sure to include a link. 

When I visit http://blucka12.github.io/spring2014/ I see both of these posts missing their authors and with incorrect dates.  @gerbal's got the right idea on how to fix the author and the dates are part of the filename.

Blucka12 committed:
Oak corrected the "author" line in the post

Changed it from caps to lowercase as it is in the config_yml

Blucka12 committed:
Oak corrected the "author" line in second post

Again with the lowercase again, and attempt #2 to have a decent looking numbered list...

Blucka12 committed:
Oak added turtle image to the assignment

And thus it moves forward...

Blucka12 committed:
Oak added turtle image to the assignment correctly

I found the "preview" button.

gerbal commented on _posts/2014-01-17-blucka12turtle.md:
```diff
@@ -0,0 +1,53 @@
+---
+format: post
```
This should be `layout: post`

Blucka12 committed:
Oak changed "format:post" to "layout:post"

Ah, yes.

eah13 commented:
@Blucka12 see if you can get this post to have the right date (this will mean changing the filename) and the code block to show up correctly before we chat this PM.

Blucka12 commented:
I'm unclear on what date should be on the post. The "Turtle Assignment" was posted on the 17th before midnight, but the other stuff came later, methinks.

When I "preview" the code block, it appears the way I feel others show up, but after the commit, it seems to get wonky...not sure what I'm getting wrong, there...

Blucka12 commented:
I'm trying to delete "OakPythonExercise.md", but am not able to get to that option...

That was just practice we were doing in class that I mistakenly made into the pull request...

gerbal commented:
@Blucka12 you already deleted it in commit b3159a6

Blucka12 committed:
Oak edited the date that the assignment was posted

...and think it's right

Blucka12 committed:
Oak is testing a potential fix

Blucka12 committed:
Oak edited the assignment

Added spaces and hashtags

Blucka12 committed:
Oak added output for ex #1

Blucka12 committed:
Oak edited #5 and added Outputs for the rest
